gizmo.py

```diff
@@ -0,0 +1,93 @@
+import numpy
+import pandas
+
+def hello(name, country="Finland"):
+    
+    print("Hello %s, how are things in %s?" % (name, country))
```

Comment:
alternatively:

```python
print(f"Hello {name}, how are things in {country}?")
```

this uses python [f-strings](https://realpython.com/python-f-strings/), which are awesome.